test/tests/functional/pbs_validate_job_qsub_attributes.py

```diff
@@ -93,8 +93,13 @@ def test_qsub_N_cmdline(self):
         This test case validates job name passed via, qsub -N command line
         """
         J = Job(TEST_USER, attrs={ATTR_N: 'j&whoami&gt;/tmp/b&'})
-        jid = self.server.submit(J)
-        self.logger.info('Successfully created job: ' + jid)
+        try:
+            jid = self.server.submit(J)
+        except PbsSubmitError as e:
+            self.assertTrue("illegal -N value" in e.msg[0])
+            self.logger.info('qsub: illegal -N value.Job not submitted')
+        else:
```

Comment:
Thanks for addressing, One question I have in else part. 

Basically, we are expecting a "qsub" to fail to submit a JOB with an illegal value. 
So, we should submit the JOB and expect msg in the exception. I think **else** part is not required here.